### PR TITLE
Decision to WorkflowTask or Command renaming

### PR DESCRIPTION
**Pull request by mfateev**

### CODE REVIEW

File: src/main/java/io/temporal/internal/metrics/MetricsType.java [hunk 1]
```diff
@@ -66,15 +60,15 @@
   public static final String ACTIVITY_E2E_LATENCY =
       TEMPORAL_METRICS_PREFIX + "activity_endtoend_latency";
   public static final String ACTIVITY_TASK_ERROR_COUNTER =
-      TEMPORAL_METRICS_PREFIX + "activity_task_error";
+      TEMPORAL_METRICS_PREFIX + "activity_task_error_thrown";
```
Comment (samarabbas) [inline src/main/java/io/temporal/internal/metrics/MetricsType.java:63]: can we just call it activity_task_error? _thrown suffix won't be meaningful for go-sdk.

File: src/main/java/io/temporal/internal/metrics/MetricsType.java [hunk 2]
```diff
@@ -66,15 +60,15 @@
...
   public static final String LOCAL_ACTIVITY_TOTAL_COUNTER =
       TEMPORAL_METRICS_PREFIX + "local_activity_total";
   public static final String LOCAL_ACTIVITY_CANCELED_COUNTER =
       TEMPORAL_METRICS_PREFIX + "local_activity_canceled";
   public static final String LOCAL_ACTIVITY_FAILED_COUNTER =
       TEMPORAL_METRICS_PREFIX + "local_activity_failed";
   public static final String LOCAL_ACTIVITY_ERROR_COUNTER =
-      TEMPORAL_METRICS_PREFIX + "local_activity_panic";
+      TEMPORAL_METRICS_PREFIX + "local_activity_error_thrown";
```
Comment (samarabbas) [inline src/main/java/io/temporal/internal/metrics/MetricsType.java:71]: can we just call it local_activity_task_error? _thrown suffix won't be meaningful for go-sdk.

File: src/main/java/io/temporal/internal/sync/WorkflowThreadImpl.java [hunk 3]
```diff
@@ -55,7 +55,7 @@
   class RunnableWrapper implements Runnable {
 
     private final WorkflowThreadContext threadContext;
-    private final DecisionContext decisionContext;
+    private final ReplayWorkflowContext replayWorkflowContext;
```
Comment (samarabbas) [inline src/main/java/io/temporal/internal/sync/WorkflowThreadImpl.java:58]: why ReplyWorkflowContext?  Looks like a comment would be helpful here.

Comment (mfateev) [reply src/main/java/io/temporal/internal/sync/WorkflowThreadImpl.java:58]: Added TODO to remove from there.

**Pull request merged**